src/pybind/mgr/orchestrator_cli/module.py

```diff
@@ -622,14 +622,10 @@ def _update_mgrs(self, num=None, hosts=[], label=None):
         "name=label,type=CephString,req=false",
         'Update the number of monitor instances')
     def _update_mons(self, num=None, hosts=[], label=None):
-
-        placement = orchestrator.PlacementSpec(label=label, count=num, hosts=hosts)
-        if not hosts and not label:
+        if not num and not hosts and not label:
```

Comment:
i am looking at `PlacementSpec.validate()`, seems it's also acceptable if `self.count` is None. am i missing something?

Reply:
Right, for the use case when creating services, an empty PlacementSpec is also valid. But for the update use case, we need some form of specification, otherwise we've been called without parameters.